content/appleboy/drone-scp/index.md

```diff
@@ -0,0 +1,197 @@
+---
+date: 2017-01-06T00:00:00+00:00
+title: SCP
+author: appleboy
+tags: [ ssh, scp ]
+repo: appleboy/drone-scp
+logo: scp.svg
```

Comment:
let's use `term.svg` here (file already exists) and remove the scp logo. The scp logo in this pull request is for the SCP foundation which is not the same as the scp utility :)